lib/discourse_api/api/users.rb

```diff
@@ -75,6 +75,11 @@ def revoke_admin(user_id)
         response = put("admin/users/#{user_id}/revoke_admin")
         response[:body]
       end
+
+      def by_external_id(external_id)
```

Comment:
they added this call in the new version. I replaced the `user_external_id` call used in our app.